Doc/faq/windows.rst

```diff
@@ -329,17 +329,11 @@ Prior to Python 2.7 and 3.2, to terminate a process, you can use :mod:`ctypes`::
 
 In 2.7 and 3.2, :func:`os.kill` is implemented similar to the above function,
 with the additional feature of being able to send :kbd:`Ctrl+C` and :kbd:`Ctrl+Break`
-to console subprocesses which are designed to handle those signals. See
+to console subprocesses which are designed to handle those signals.  See
 :func:`os.kill` for further details.
 
 How do I extract the downloaded documentation on Windows?
 ---------------------------------------------------------
 
-Sometimes, when you download the documentation package to a Windows machine
-using a web browser, the file extension of the saved file ends up being .EXE.
-This is a mistake; the extension should be .TGZ.
-
-Simply rename the downloaded file to have the .TGZ extension, and WinZip will be
-able to handle it.  (If your copy of WinZip doesn't, get a newer one from
-https://www.winzip.com.)
-
+Use 7zip or your favorite archive program to handle the downloaded documentation
+package.  If not look for a newer version that does.
```

Comment:
This new wording seems okay to me.  Windows built-in archive support cannot handle .tgz.  7zip is probably the preferred tool but any decent archive program like WinZip or WinRAR should handle it too.  The note about .EXE extensions seems unnecessary.  I've never seen a Windows browser do that but maybe they did at some point.  I'd guess that is useless historical trivia at this point.